nvflare/fuel/utils/validation_utils.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def check_positive_int(name, value):
+    if not isinstance(value, int):
+        raise TypeError(f"{name} must be an int, but got {type(value)}.")
+    if value <= 0:
+        raise ValueError(f"{name} must > 0, but got {value}")
+
+
+def check_whole_int(name, value):
+    if not isinstance(value, int):
+        raise TypeError(f"{name} must be an int, but got {type(value)}.")
+    if value < 0:
+        raise ValueError(f"{name} must >= 0, but got {value}")
```

Comment:
suggest "check_non_negative_int" if we want this method.
As integer can be positive or negative.